website/docs/configuration/recommended-rbac-configuration.mdx

```diff
@@ -178,3 +180,142 @@ roleRef:
+
+| Provider          | API Group                 | Resource        |
+| ---               | ---                       | ---             |
+| AppMesh           | appmesh.k8s.aws           | virtualnode     |
+|                   | appmesh.k8s.aws           | virtualrouter   |
+|                   | appmesh.k8s.aws           | virtualservice  |
+| Linkerd           | split.smi-spec.io         | trafficsplit    |
+| Istio             | networking.istio.io       | destinationrule |
+|                   | networking.istio.io       | virtualservice  |
+| Contour           | projectcontour.io         | httpproxy       |
+| Gloo              | gateway.solo.io           | routetable      |
+|                   | gloo.solo.io              | upstream        |
+| Nginx             | networking.k8s.io         | ingress         |
+| Skipper           | networking.k8s.io         | ingress         |
+| Traefik           | traefik.containo.us       | traefikservice  |
+| Open Service Mesh | split.smi-spec.io         | trafficsplit    |
+| Kuma              | kuma.io                   | trafficroute    |
+| GatewayAPI        | gateway.networking.k8s.io | httproute       |
+
+For example, the following manifest shows how `gitops-canaries-reader` has been extended to allow the user for viewing TrafficSplit resources when Linkerd is used:
+
+```yaml title="gitops-canaries-reader.yaml"
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: gitops-canaries-reader
+rules:
+- apiGroups:
+  - flagger.app
+  resources:
+  - canaries
+  - metrictemplates
+  verbs:
+  - get
+  - list
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - get
+  - watch
+  - list
+# Additional permissions for Linkerd resources are added below
+- apiGroups:
+  - split.smi-spec.io
+  resources:
+  - trafficsplits
+  verbs:
+  - get
+  - list
+```
+
+### Setting up remote cluster permissions
+
+In order to be able to view remote cluster information, such as canaries in a remote cluster, from the management cluster, you need to take into account the following:
+- The service account that is used to access the remote cluster, needs to be able to list the namespaces and custom resource definitions of its cluster. In addition to those permissions, it also needs to be able to impersonate users and groups.
+- The user or group that logs in to the management cluster, needs appropriate permissions to certain resources of the remote cluster.
+
+For example, the following manifest ensures that the `wego-admin` user will be able to view canary information from a remote cluster:
```

Comment:
Assume this manifest would need to be applied on the remote cluster? If Enterprise has a preferred way the user does this we should advise on the approach

Reply:
Not sure if there is a preferred way, cc @foot for input. This approach was tested with non-CAPI clusters, maybe there's a way to apply it for CAPI clusters.